keras/backend/numpy/image.py

```diff
@@ -13,6 +13,22 @@
 )
 
 
+def psnr(image1, image2, max_val):
+    if image1.shape != image2.shape:
+        raise ValueError(
+            "Image shapes must match for PSNR calculation. "
+            f"Received shapes: image1.shape={image1.shape}, "
+            "image2.shape={image2.shape}. "
+        )
+
+    max_val = np.float32(max_val)
+    image1 = image1.astype(np.float32)
+    image2 = image2.astype(np.float32)
+    mse = np.mean(np.square(image1 - image2), axis=(-3, -2, -1))
+    psnr = 20 * np.log(max_val) / np.log(10.0) - (10 / np.log(10)) * np.log(mse)
```

Comment:
Instead of dividing by log(10) you should use the `log10` operator everywhere. I think it's available in all backends.

Reply:
Ok will change that!